numba/cuda/cudadrv/drvapi.py

```diff
@@ -27,330 +27,354 @@
-'cuOccupancyMaxActiveBlocksPerMultiprocessor': (c_int,
-                                                POINTER(c_int), cu_function, c_size_t, c_uint),
-
-# CUresult CUDAAPI cuOccupancyMaxActiveBlocksPerMultiprocessorWithFlags(int *numBlocks,
-#                                                              CUfunction func,
-#                                                              int blockSize,
-#                                                              size_t dynamicSMemSize,
-#                                                              unsigned int flags);
-'cuOccupancyMaxActiveBlocksPerMultiprocessor': (c_int,
-                                                POINTER(c_int), cu_function, c_size_t, c_uint),
-
-# CUresult CUDAAPI cuOccupancyMaxPotentialBlockSize(int *minGridSize, int *blockSize,
-#                                                   CUfunction func, CUoccupancyB2DSize blockSizeToDynamicSMemSize,
-#                                                   size_t dynamicSMemSize, int blockSizeLimit);
-'cuOccupancyMaxPotentialBlockSize': (c_int,
-                                     POINTER(c_int), POINTER(c_int), cu_function, cu_occupancy_b2d_size, c_size_t, c_int),
-
-# CUresult CUDAAPI cuOccupancyMaxPotentialBlockSizeWithFlags(int *minGridSize, int *blockSize,
-#                                                            CUfunction func, CUoccupancyB2DSize blockSizeToDynamicSMemSize,
-#                                                            size_t dynamicSMemSize, int blockSizeLimit, unsigned int flags);
-'cuOccupancyMaxPotentialBlockSizeWithFlags': (c_int,
-                                              POINTER(c_int), POINTER(c_int), cu_function, cu_occupancy_b2d_size, c_size_t, c_int, c_uint),
-
-# CUresult cuIpcGetMemHandle ( CUipcMemHandle* pHandle, CUdeviceptr dptr )
-'cuIpcGetMemHandle': (c_int,
-                      POINTER(cu_ipc_mem_handle), cu_device_ptr),
-
-# CUresult cuIpcOpenMemHandle ( CUdeviceptr* pdptr, CUipcMemHandle handle, unsigned int  Flags )
-
-'cuIpcOpenMemHandle': (c_int,
-                       POINTER(cu_device_ptr), cu_ipc_mem_handle, c_uint),
-
-# CUresult cuIpcCloseMemHandle ( CUdeviceptr dptr )
-
-'cuIpcCloseMemHandle': (c_int,
-                        cu_device_ptr),
-
-# CUresult cuCtxEnablePeerAccess ( CUcontext peerContext, unsigned int  Flags )
-'cuCtxEnablePeerAccess': (c_int,
-                          cu_context, c_int),
-
-# CUresult cuDeviceCanAccessPeer ( int* canAccessPeer,
-#                                  CUdevice dev, CUdevice peerDev )
-'cuDeviceCanAccessPeer': (c_int,
-                          POINTER(c_int), cu_device, cu_device),
+    # CUresult cuMemcpyHtoD(CUdeviceptr dstDevice, const void *srcHost,
+    #                       size_t ByteCount);
+    'cuMemcpyDtoD': (c_int, cu_device_ptr, cu_device_ptr, c_size_t),
+
+    # CUresult cuMemcpyHtoDAsync(CUdeviceptr dstDevice, const void *srcHost,
```

Comment:
```suggestion
    # CUresult cuMemcpyDtoDAsync(CUdeviceptr dstDevice, const void *srcHost,
```

Reply:
Good catch - I think it's not just the name that is wrong, but also the parameters so I'll adjust this then push.

Reply:
Should now be correct.

Reply:
Is this pushed up?

Reply:
I pushed to the wrong branch. This should be pushed now.